index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+  <nav>
+    <div class="nav-bar">
+      <h1 class="logo">Kajal</h1>
+      <img src="./img/hambug.png" alt="hamburger" class="hambug">
+    </div>
+  </nav>
+
+  <header>
+
+    <div class="container">
+
+      <h1 class="header-title">I'm Kajal Pramanik to see you!</h1>
+      
+      <p class="header-text">
+        I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+      </p>
+      
+      <p class="header-social-text">Let's connect</p>
+      
+      <img src="./img/social-media.png" alt="social-icon" class="social-icon">
+      
+    </div>
```

Comment:
- [ ] This is the headline section so it would be better if you change this `<div>` to `<section>` so that it will easy to read and also improve the SEO